homeassistant/components/decora_wifi/__init__.py

```diff
@@ -1 +1,100 @@
 """The decora_wifi component."""
+
+import logging
+
+from decora_wifi import DecoraWiFiSession
+from decora_wifi.models.iot_switch import IotSwitch
+from decora_wifi.models.permission import Permission
+from decora_wifi.models.person import Person
+from decora_wifi.models.residence import Residence
+from decora_wifi.models.residential_account import ResidentialAccount
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_PASSWORD,
+    CONF_USERNAME,
+    EVENT_HOMEASSISTANT_STOP,
+    Platform,
+)
+from homeassistant.core import Event, HomeAssistant
+from homeassistant.exceptions import ConfigEntryError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+PLATFORMS = [Platform.LIGHT]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
A thing that changes over the last year is that we now store runtime data in `entry.runtime_data` rather than in `hass.data[DOMAIN][entry.entry_id]`. We can extend the type of the configEntry to set the type of that field.

Checkout https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/runtime-data